src/coreclr/vm/amd64/JitHelpers_Fast.asm

```diff
@@ -279,7 +279,7 @@ ifdef WRITE_BARRIER_CHECK
         ; Check that our adjusted destination is somewhere in the shadow gc
         add     r10, [g_GCShadow]
         cmp     r10, [g_GCShadowEnd]
-        ja      NoShadow
```

Comment:
Not related to this change, but I wonder if the GC shadow thing is actually usable/useful. 
I see that we fix things in both CoreClr and NativeAOT, but never saw GC shadow actually enabled and used. Even when we need to investigate something, we always use something else, for many releases. What this code is waiting for?